padertorch/train/hooks.py

```diff
@@ -1017,12 +1031,21 @@ def get_optimizer(self, trainer):
         return optimizer.optimizer
 
     def get_value(self, trainer):
-        return self.get_optimizer(trainer).param_groups[0]['lr']
+        opt = self.get_optimizer(trainer)
+        lrs = [param_group['lr'] for param_group in opt.param_groups]
+        if len(set(lrs)) == 1:
```

Comment:
Why returning a scalar, when all lrs are the same?
Why not simply `if len(lrs) == 1:`?